aquadoggo/src/materializer/worker.rs

```diff
@@ -495,10 +495,8 @@ where
                         }
                     }
 
-                    // Trigger status update when successful
-                    if result.is_ok() {
-                        on_complete(item.input());
-                    }
+                    // Trigger removing the task from the task store
+                    on_complete(item.input());
```

Comment:
I made this change so tasks that fail silently don't get rescheduled after a node restart.